match.py

```diff
@@ -51,7 +51,11 @@ def do_match(c):
                 household_project.upload_clks(
                     system, c.get_household_clks_raw(system, project_name)
                 )
-            household_project.start_run(c.matching_threshold)
+            if type(c.matching_threshold) == list:
+                threshold = c.matching_threshold[0]
```

Comment:
One last thing, we should also split out the household matching threshold into its own config value rather than just pick the first one from a list. It's a little surprising to say "these thresholds align with the projects listed above" but then one of them also has a second meaning

Reply:
For now, I think that this makes sense, but long term I think from an ease of use perspective, we might want to make configuring household and non-household runs a more unified process. To me, the fact that the code essentially disregards some configuration info if other info has a certain value complicates the process for, users as they might be confused as to what matching threshold applies to a given run seeing as we're introducing a second place that threshold numbers can go. 

To help clear this up in the short term, i think we should rename matching_threshold to something like `"individual_matching_threshold"` (or just `"individual_threshold"` if that's too long); I could implement that very quickly without changing much else if I kept the name of the property defined by the `@property` decorator the same, but doing so might confuse developers. What do you think, @dehall?

Reply:
Yeah I agree, there are definitely ways it could be cleaned up for consistency. Let's add this as an item to the backlog though as I think it needs some more consideration on what we may want to do in the future. For example one possible approach would be to use 2 separate config files, one for individuals and one for households, but then that has its own downsides of the user has to keep the common items in sync, so I'd want to really talk more about that before we change too much.